src/constants/misc.ts

```diff
@@ -17,9 +17,6 @@ export const L2_DEADLINE_FROM_NOW = 60 * 5
 export const DEFAULT_TXN_DISMISS_MS = 25000
 export const L2_TXN_DISMISS_MS = 5000
 
-// threshold for when to show a price changed warning
-export const DEFAULT_PRICE_CHANGED_THRESHOLD = 0.01
-
```

Comment:
accidentally added this early in diff PR and ended up not needing it in this PR whoops